contributors/devel/sig-testing/bazel.md

```diff
@@ -37,7 +37,7 @@ $ bazel test //pkg/kubectl/...
 
 [Prow]: https://prow.k8s.io
 [bazel-install]: https://www.bazel.io/versions/master/docs/install.html
-[kazel]: https://git.k8s.io/repo-infra/kazel
+[kazel]: https://github.com/kubernetes/repo-infra/tree/master/cmd/kazel
```

Comment:
Please still use the https://git.k8s.io url shortener if updating links. For a bit more info, check out the [link section](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#links) of the [style guide](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md).